### PR TITLE
Add store getters as second argument to store.watch() computed fn (solve #460)

### PR DESCRIPTION
**Pull request by LinusBorg**
* the first parameter of store.watch(), which is a function, now receives the store.getters object as its second argument.
* added unit test

Fullfills request #460

### CODE REVIEW

File: test/unit/test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Vuex, { mapState, mapMutations, mapGetters, mapActions } from '../../dist
 Vue.use(Vuex)
 
 const TEST = 'TEST'
-const TEST2 = 'TEST2'
+// const TEST2 = 'TEST2'
```
Comment (LinusBorg) [inline test/unit/test.js]: eslint complained that this constant was never used anywhere in the tests so i commented it out. Should I remove it, or is it needed for something? Tests all worked without it.

Comment (ktsn) [inline test/unit/test.js]: I think we can remove it.

**Comment by LinusBorg** [comment]
@ktsn removed it.

**Comment by yyx990803** [comment]
@LinusBorg looks like the dist file is still included in the PR

**Comment by LinusBorg** [comment]
Oh right, sorry

**Comment by LinusBorg** [comment]
@yyx990803 I reversed the changes to /dist.

**Pull request merged**